plugins/CoreHome/vue/src/ReportingMenu/ReportingMenu.vue

```diff
@@ -0,0 +1,359 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div class="reportingMenu">
+    <ul
+      class="navbar hide-on-med-and-down"
+      role="menu"
+      :aria-label="translate('CoreHome_MainNavigation')"
+    >
+      <li
+        class="menuTab"
+        role="menuitem"
+        v-for="category in menu"
+        :class="{ 'active': category.id === activeCategory }"
+        :key="category.id"
+      >
+        <a
+          class="item"
+          tabindex="5"
+          href=""
+          @click.prevent="loadCategory(category)"
+        >
+          <span
+            :class="`menu-icon ${category.icon ? category.icon : 'icon-arrow-right'}`"
+          />
+          {{ category.name }}
```

Comment:
The additional space between the icon and the menu title actually causes the "regression" that can be seen in the UI file changes. Long menu items are now always wrapped into a new line after the icon. This should fix that:
```suggestion
          />{{ category.name }}
```

Reply:
Applied.